### ISSUE TITLE
Dropdown issue in navbar

### ISSUE THREAD

**Issue by andreimladin** [creativetimofficial/material-kit#41]
I've just bought the bootstrap material pro kit and I have an issue trying to integrate it with angular2.

It doesn't work even if I added the arrive.js library.

And also I receive this in console when adding the nouislider.min.js:

> nouislider.min.js:10 Uncaught TypeError: Cannot set property 'Link' of undefined(anonymous function) @ nouislider.min.js:10800 @ nouislider.min.js:10
> **webpack_require** @ bootstrap eaef8bc…:
> 500 @ main.browser.ts:65__webpack_require__ @ bootstrap eaef8bc…:
> 50webpackJsonpCallback @ bootstrap eaef8bc…:
> 21(anonymous function) @ main.bundle.js:2

I would much appreciate you helping me to figure out the problem. Thanks

**Comment by alexandru-paduraru** [comment]
@andreimladin thank you for using our product. Is it possible to give us a link to your application? Then we will check on the live example what can be the issues and give you a solution because right now the error is not familiar to us.

Best,
Alex

**Comment by andreimladin** [comment]
I didn't publish it yet. The issue was related to bootstrap. I don't understand why it didn't work when simply I just imported the scripts via webpack.
I added the ng2-bootstrap as support and added the DROPDOWN_DIRECTIVES as directive in my component. This fixed the issue.

Maybe it would be very useful and more attractive for your clients if you would have some guidelines for integrating with other 3rd parties (like angular1&2, webpack, etc). 
I'm just saying, it's not a complain, your framework is great.

**Comment by alexandru-paduraru** [comment]
@andreimladin thank you for the information and for the suggestions. We will definitely migrate our products and make them compatible with Angular 1&2, React, Meteor, Wordpress etc. We started to migrate our [Light Bootstrap Dashboard](http://demos.creative-tim.com/light-bootstrap-dashboard/dashboard) to Angular, so we are moving slowly to that area too.

**Comment by dbarrett24** [comment]
@andreimladin, so do none of them currently work in Angular 1? I purchased the material dashboard pro theme (not the angular 2 version) in hopes of using it in an Angular 1 project. If it's not going to work, I'd like a refund.

**Comment by alexandru-paduraru** [comment]
@dbarrett24 we have 2 types of products. Simple HTML/CSS templates (like Material Dashboard PRO that you've got) and now we started to migrate them to Angular 2. Of course, it will work with Angular 1 if you add it to your project, you just need to create your own components because you've got the simple HTML dashboard. 

We have a lot of users who integrate our products in Angular, that's why we started to release the Angular 2 Versions, to make their development process easier.

Let us know your progress.

Best,
Alex
